backend/swagger/cirrusspark.yaml

```diff
@@ -1371,6 +1371,369 @@ paths:
+        required: true
+        type: "string"
+      - in: "body"
+        name: "body"
+        description: "Updated hours by input type"
+        required: true
+        schema:
+          $ref: ""
+      responses:
+        "400":
+          description: "Invalid caller email supplied"
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "200":
+          description: "OK"
+    delete:
+      tags:
+      - "loghours"
+      summary: "Deletes hours by input type belonging to {email}"
+      description: ""
+      operationId: "deleteEmployeeHoursInputType"
+      produces:
+      - "application/json"
+      parameters:
+      - name: email
+        in: "path"
+        description: "Employee email"
+        required: true
+        type: "string"
+      responses:
+        "400":
+          description: "Invalid caller email supplied"
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "200":
+          description: "OK"
+  /task:
+    post:
+      tags:
+      - "tasks"
+      summary: "Creates task"
+      description: ""
+      operationId: "createTask"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Created user object"
+        required: true
+        schema:
+          $ref: ""
+      responses:
+        "400":
+          description: "Invalid caller email supplied"
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "201":
+          description: "Created"
+    get:
+      tags:
+      - "tasks"
+      summary: "Gets all tasks"
+      description: ""
+      operationId: "getAllTask"
+      produces:
+      - "application/json"
+      responses:
+        "400":
+          description: "Invalid caller email supplied"
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "200":
+          description: "Ok"
+  /task/{taskId}:
+    get:
+      tags:
+        - "tasks"
+      summary: "Gets specific task by {id}"
+      description: ""
+      operationId: "getTask"
+      produces:
+        - "application/json"
+      parameters:
+      - name: taskId
+        in: "path"
+        description: "Task Id"
+        required: true
+        type: "string"
+      responses:
+        "400":
+          description: "Invalid caller email supplied"
+        "404":
+          description: "Not found"
+        "401":
+          description: "Unauthorized"
+        "500":
+          description: "Unexpected error"
+        "200":
+          description: "Ok"
+    post:
```

Comment:
Is it a post here?

Reply:
Yeah, but I didn't want to change the backend and break something this close to the release